mantle/platform/api/do/api.go

```diff
@@ -139,7 +145,7 @@ func (a *API) CreateDroplet(ctx context.Context, name string, sshKeyID int, user
 			Size:              a.opts.Size,
 			Image:             a.image,
 			SSHKeys:           []godo.DropletCreateSSHKey{{ID: sshKeyID}},
-			IPv6:              true,
+			IPv6:              a.opts.IsCustomImage,
```

Comment:
I would say this probably could use a comment to explain the logic here. Of course the other side of the argument to that is that one could always use `git blame`.